pytest_selenium/drivers/cloud.py

```diff
@@ -30,13 +30,14 @@ def config(self):
         config.read([name, os.path.join(os.path.expanduser('~'), name)])
         return config
 
-    def get_credential(self, key, env):
-        try:
-            value = self.config.get('credentials', key)
-        except (configparser.NoSectionError,
-                configparser.NoOptionError,
-                KeyError):
-            value = os.getenv(env)
-        if not value:
-            raise MissingCloudCredentialError(self.name, key, env)
-        return value
+    def get_credential(self, key, envs):
+        for env in envs:
```

Comment:
You're the boss. If you want me to handle the case where it is a string let me know how. :)

My .02 are however that if you're going to call this directly you'll also be able to figure out that what you send in needs to be in the form of a list. 👍